db-scheduler/src/main/java/com/github/kagkarlsson/scheduler/SchedulerBuilder.java

```diff
@@ -108,6 +104,11 @@ public SchedulerBuilder executorService(ExecutorService executorService) {
         return this;
     }
 
+    public SchedulerBuilder executorServiceShutdownTimeout(Duration shutdownTimeout) {
```

Comment:
Not sure about the name. Especially since it is not applicable for the builder-method `executorService` where you send in the externally managed one.

What about `shutdownTimeout`?